src/vmm/src/gdb/target.rs

```diff
@@ -377,8 +386,13 @@ impl MultiThreadBase for FirecrackerTarget {
         let data_len = data.len();
         let vcpu_state = &self.vcpu_state[tid_to_vcpuid(tid)];
 
+        let vmm = &self.vmm.lock().map_err(|_| {
+            error!("Error locking vmm in read addr");
+            TargetError::Fatal(GdbTargetError::VmmLockError)
+        })?;
```

Comment:
Maybe we should just `unwrap` on `lock`. I feel if this even occurs this just means we have a bug somewhere, so it is better to panic. Also there are already places where we just `unwrap`.

Reply:
Moved them all to unwraps now